volatility3/framework/plugins/windows/netscan.py

```diff
@@ -161,7 +161,7 @@ def determine_tcpip_version(
             raise NotImplementedError(
                 "Kernel Debug Structure version format not supported!"
             )
-        except:
+        except:  # noqa: E722
```

Comment:
This is exactly the type of thing *not* to ignore.  It should be fixed ideally.  I can ask the guy who wrote this what he was expecting to catch here...

Reply:
Yeah, sadly I don't know either (and that's why I wish the original author had put the list in here).  I guess either `Exception` (which the linter may complain about separately) or Just `NotImplementedError` and the classic `InvalidAddressException` since those the the most common one?  Then add a `FIXME` to tell devs to add any additional exceptions that are hit?

Reply:
I guess we'll keep the ignore in here, but could you add `FIXME: ` to one of the comments below please, just so we don't lose it again?  That should be it save for the reviews from the rest of the team...